pkg/kademlia/kademlia_test.go

```diff
@@ -39,19 +39,21 @@ func TestNewKademlia(t *testing.T) {
 	}{
 		{
 			id: func() *node.ID {
-				id, err := node.NewID()
+				id, err := node.NewFullIdentity(ctx, 12, 4)
```

Comment:
Could we have a separate function `NewTestingIdentity`, which sets the difficulty params correctly for testing, rather than specifying them everywhere.

Reply:
👍